packages/core/src/views/account-center/Maximized.svelte

```diff
@@ -229,16 +216,16 @@
   }
 
   .app-button {
+    /* font-family: var(--account-center-app-btn-font-family, inherit); */
```

Comment:
```suggestion
    font-family: var(--account-center-app-btn-font-family, inherit);
```
this was a requested variables so I would imagine it is in use

Reply:
I'm not sure about this one. I REALLY don't want it, but let me look.